mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -864,6 +891,14 @@
             <WeaponArray Link="MothershipCoreWeapon" Turret="Nexus"/>
         </Modification-->
     </CBehaviorBuff>
+    <CBehaviorBuff id="BatteryAcquireTargetCooldown">
+        <InfoFlags index="Hidden" value="1"/>
+        <EditorCategories value="AbilityorEffectType:Units"/>
+        <MaxStackCount value="65535"/>
+        <MaxStackCountPerCaster value="1"/>
+        <TimeScaleSource Value="Caster"/>
+        <Duration value="0.063"/>
+    </CBehaviorBuff>
```

Comment:
Fixed an issue where repeatedly issuing a Stop command could increase the shield regen rate of a Shield Battery's target

The behavior that indicates that the Shield Battery is on cooldown. E.g. auto-cast requires this to be not present anymore

The duration of 0.63 is odd. Is this preventing the cast for 1 or 2 game updates? Should always be multiples of 0.0625 for clarity